device/dhcp/common/parameters.yaml

```diff
@@ -0,0 +1,141 @@
+components:
+  schemas:
+    Device.DhcpParams:
```

Comment:
This again looks like duplicate with v4 params. Please keep only common if possible reused by both or include this   in both with extra params if any .